### PR TITLE
protein-translation: Remove repetition from canonical data

### PR DESCRIPTION
**Pull request by sjwarner-bp**
I noticed there was some redundancy in the test names - removed it :slightly_smiling_face:

### CODE REVIEW

**Comment by sjwarner-bp** [comment]
I wasn't sure if it was a big enough change to bother updating the version number or not - I'd be interested to hear thoughts.

File: exercises/protein-translation/canonical-data.json [hunk 1]
```diff
@@ -136,13 +136,13 @@
...
         {
-          "description": "Translation stops if STOP codon in middle of sequence of three-codon sequence",
+          "description": "Translation stops if STOP codon in middle of three-codon sequence",
           "property": "proteins",
           "strand": "UGGUAGUGG",
           "expected": ["Tryptophan"]
         },
         {
-          "description": "Translation stops if codon in middle of sequence of six-codon sequence",
+          "description": "Translation stops if codon in middle of six-codon sequence",
```
Comment (ErikSchierboom) [inline exercises/protein-translation/canonical-data.json]: Shouldn't "Translation stops if codon" be "Translation stops if STOP codon"

Comment (sjwarner-bp) [reply exercises/protein-translation/canonical-data.json]: That is indeed the case - good spot! I'll fix that one too :slightly_smiling_face:

**Comment by coriolinus** [comment]
@sjwarner-bp re: increment the version?

It's a matter of discretion, really. The [Semantic Version 2.0 spec](http://semver.org/) says that the PATCH field is for "backwards-compatible bug fixes"; this is certainly backwards-compatible, but is it a bug fix? 

My instinct is to say that as this change might change the output of exercise generators, you may as well update the patch field of the version. However, I don't feel strongly about this at all; I'm happy to leave it to your discretion.

**Comment by sjwarner-bp** [comment]
On it :+1:

**Pull request merged**